build.gradle.kts

```diff
@@ -6,6 +6,10 @@ plugins {
   idea
 }
 
+springBoot {
+  mainClass.value("uk.gov.justice.digital.hmpps.visitscheduler.VisitScheduler")
```

Comment:
should this be VisitSchedulerApplication or VisitSchedulerApplicationKt I have no idea but it does say mainClass,.. so I'm assuming it's VisitScheduler, only one way to find out

Reply:
found out it was the file not the class bizzar format